leaf-ui/js/onFunctions.js

```diff
@@ -942,7 +942,12 @@ paper.on({
                 if (cell instanceof joint.shapes.basic.Actor) {
                     actorInspector.render(cell);
                 } else {
-                    elementInspector.render(cell);
+                    var elementInspector = new ElementInspector({model: cell});
+                    console.log(elementInspector)
+                    // HTML moved here from initializeElements
+                    $('.inspector').append(elementInspector.el);
+                    elementInspector.switchModel(cell)
```

Comment:
You shouldn't need a switchModel method here since you already passed the cell in as the model in line 945